sorts/sorts_case_test.go

```diff
@@ -13,25 +13,46 @@ type sortTest struct {
 	name     string
 }
 
-var arr []int = makeRandArray(500_000)
+var uarr []int = makeRandomUnsignedSlice(500_000)
+var arr []int = makeRandomSignedSlice(500_000)
 
 var sortTests = []sortTest{
 	//Sorted slice
 	{[]int{1, 2, 3, 4, 5, 6, 7, 8, 9, 10},
-		[]int{1, 2, 3, 4, 5, 6, 7, 8, 9, 10}, "Sorted"},
+		[]int{1, 2, 3, 4, 5, 6, 7, 8, 9, 10}, "Sorted Signed"},
```

Comment:
The test case names seem mixed. These are unsigned and the ones with negative values will be signed. Please fix that.

Reply:
haha don't know what I was thinking, fixed it now